src/lib/shared/components/ui/Jumbotron/Jumbotron.tsx

```diff
@@ -1,32 +1,66 @@
-import { Box } from "@mui/material";
+import { Box } from '@mui/material';
 import { SxProps, Theme } from '@mui/material/styles';
 import { Button } from '../Button';
 
 import { styled } from '@mui/material/styles';
 import { H1, Paragraph } from '../Typography/';
+import { CelebrationContainer } from '../Containers';
 
 export const TEST_IDS = {
     CONTAINER: 'container',
+    CELEBRATION: 'celebration',
 } as const;
 
+interface BackgroundImage {
+    backgroundImageUrl: string;
+    useCelebrationStyling?: false;
+}
+interface CelebrationStyling {
+    backgroundImageUrl?: never;
+    useCelebrationStyling: true;
+}
+
```

Comment:
``` suggestion
interface BackgroundImage {
    backgroundImageUrl: string;
    useCelebrationStyling?: never;
}
interface CelebrationStyling {
    backgroundImageUrl?: never;
    useCelebrationStyling: true;
}

type BackgroundProps = BackgroundImage | CelebrationStyling;
```

This will require `backgroundProps` to either be `{ backgroundImageUrl:'...'}` or `{ useCelebrationStyling: true}`

Reply:
I also think `withCelebrationStyling` sounds a little cleaner than `useCelebrationStyling` so we can reserve "use" for hook names